libs/application/templates/complaints-to-althingi-ombudsman/src/fields/ComplaintOverview/ComplainedFor.tsx

```diff
@@ -0,0 +1,94 @@
+import { ReviewGroup } from '@island.is/application/ui-components'
+import { GridColumn, GridRow } from '@island.is/island-ui/core'
+import React, { FC } from 'react'
+import { ComplaintsToAlthingiOmbudsman } from '../../lib/dataSchema'
+import {
+  complainedFor as complainedForMessages,
+  complaintOverview,
+} from '../../lib/messages'
+import { ComplainedForTypes } from '../../shared'
+import { mapComplainedForToMessage } from '../../utils'
+import { ValueLine } from './ValueLine'
+
+type Props = {
+  complainedForType: ComplainedForTypes
+  complainedFor: ComplaintsToAlthingiOmbudsman['complainedForInformation']
+  connection: string
+}
+
+export const ComplainedFor: FC<Props> = ({
```

Comment:
There's been some discussion about about the FC interface for react components, try not to use it unless the component is supposed to accept children. Otherwise you should declare it as a simple function component:

`export const ComplainedFor = (props: Props) => {`

Reply:
Roger - didn't know there were issues with FC